parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -47,21 +48,33 @@
 import org.apache.parquet.schema.MessageType;
 
 import static org.apache.iceberg.parquet.ParquetConversions.fromParquetPrimitive;
+import static org.apache.iceberg.util.BinaryUtil.truncateBinaryMax;
+import static org.apache.iceberg.util.BinaryUtil.truncateBinaryMin;
+import static org.apache.iceberg.util.UnicodeUtil.truncateStringMax;
+import static org.apache.iceberg.util.UnicodeUtil.truncateStringMin;
 
 public class ParquetUtil {
   // not meant to be instantiated
   private ParquetUtil() {
   }
 
   public static Metrics fileMetrics(InputFile file) {
+    return fileMetrics(file, TableProperties.WRITE_METADATA_TRUNCATE_BYTES_DEFAULT);
```

Comment:
This methods is only called by tests, so it can be made package-private. There is one use in tests outside of this package, in Spark's `TestParquetScan`. That can be replaced with `.withRecordCount(100)` because stats aren't used in that test.